x-pack/plugins/ml/public/embeddables/types.ts

```diff
@@ -57,6 +61,10 @@ export interface MlEmbeddableBaseApi<StateType extends object = object>
     PublishesDataViews,
     PublishesUnifiedSearch {}
 
+export interface MlEmbeddableBasicApi<StateType extends object = object>
+  extends DefaultEmbeddableApi<StateType>,
+    PublishesTimeRange {}
+
```

Comment:
we already have `MlEmbeddableBaseApi`
```suggestion
```

Reply:
See [explanation](https://github.com/elastic/kibana/pull/180544#discussion_r1571259233) below